src/main/resources/org/broadinstitute/hellbender/utils/plotter/AlleleFractionResultsPlotting.R

```diff
@@ -0,0 +1,126 @@
+	segments = read.table(segments_file, sep="\t", stringsAsFactors=FALSE, header=TRUE, check.names=FALSE)
+
+
+	snpContig = snp_counts[,"CONTIG"]
+    snpContig[snpContig=="X"]=23
+    snpContig[snpContig=="Y"]=24
+    snp_counts[,"CONTIG"]=as.numeric(snpContig)
+
+    segments[segments["Chromosome"]=="X","Chromosome"]=23
+    segments[segments["Chromosome"]=="Y","Chromosome"]=24
+    segments[,"Chromosome"]=as.numeric(segments[,"Chromosome"])
+
+    #exclude segments with no SNPS
+    segments = segments[segments$Num_SNPs > 0,]
+
+    plot_file_name = file.path( output_dir, paste(sample_name, "_FullGenome", ".png", sep=""))
+    png(plot_file_name, 12, 7, units="in", type="cairo", res=300, bg="white")
+    SetUpPlot("Allele Fraction", "Chromosome", TRUE)
+    AlleleFractionPlot(snp_counts, segments)
+    dev.off()
+}
+
+## Chromosome Background for data
+SetUpPlot = function(y.lab, x.lab, label_chromosomes){
+  suppressWarnings( par( mar=c(3.1,3.6,0.1,0),mgp=c(2,-0.2,-1.1) ) )
+  plot(0, type = "n", bty = "n", xlim = c(0, 1), ylim = c(0, 1),
+       xlab = "", ylab = "", main = "", xaxt = "n")
+  mtext(side=1, line=1.5, x.lab, cex=0.75, outer=FALSE)
+  mtext(side=2.2, line=1.5, y.lab, cex=0.75, las=FALSE, outer=FALSE)
+
+  if(label_chromosomes){
+  	labels = c(1:num_chromosomes)
+  	odd_indices = labels %% 2 == 1
+  	mtext(text = labels[odd_indices], side = 1, line = -0.45, at = chromosome_centers[odd_indices],
+    		las = 1, cex = par("cex.axis") * par("cex") * 0.7)
+  	mtext(text = labels[!odd_indices], side = 1, line = 0, at = chromosome_centers[!odd_indices],
+      	las = 1, cex = par("cex.axis") * par("cex") * 0.7)
+  }
+
+  for (i in 1:num_chromosomes) {
+    use.col = ifelse(i%%2 == 1, "grey90", "white")
+    rect(xleft = chromosome_starts[i], ybottom = 0, xright = chromosome_ends[i], ytop = 1, col = use.col, border = NA)
+
+    #draw vertical lines to gate off centromeres
+    lines(y = c(0, 1), x = rep(centromere_starts[i], 2), lty = 3, lwd = 0.5)
+    lines(y = c(0, 1), x = rep(centromere_ends[i], 2), lty = 3, lwd = 0.5)
+  }
+}
+
+AlleleFractionPlot = function(snp_df, segments_df){
+  for( s in 1:nrow(segments_df)) {
+     snp_indices=which(snp_df[,"CONTIG"]==segments_df[s,"Chromosome"] & snp_df[,"POS"] > segments_df[s,"Start"] & snp_df[,"POS"] < segments_df[s,"End"])
+     write.table(snp_indices, "/Users/davidben/Desktop/seg_index.txt")
+     genomic_coordinates = chromosome_starts[snp_df[snp_indices,"CONTIG"]] + snp_df[snp_indices,"POS"] / genome_length
+     ref_counts = snp_df[snp_indices, "REF_COUNT"]
+     alt_counts = snp_df[snp_indices, "ALT_COUNT"]
+     MAF = ifelse(alt_counts<ref_counts, alt_counts/(alt_counts+ref_counts), ref_counts/(alt_counts+ref_counts))
+
+     colors=c("coral", "dodgerblue")
+     points(x=genomic_coordinates, y=MAF, col=colors[s%%2+1], pch=16, cex=0.2)
+     lines(x=range(genomic_coordinates), y=rep(segments_df[s, "MAF_Post_Mean"], 2), col="black", lwd=1, lty=2)
```

Comment:
maybe also plot "confidence intervals" (+/- posterior std devs)?  we can probably file an issue to construct proper confidence intervals at some point, but i think these would be useful to see at this point.